module/app/app.go

```diff
@@ -82,6 +82,7 @@ import (
 	"github.com/gorilla/mux"
 	gravityparams "github.com/peggyjv/gravity-bridge/module/v3/app/params"
 	v2 "github.com/peggyjv/gravity-bridge/module/v3/app/upgrades/v2"
+	v3 "github.com/peggyjv/gravity-bridge/module/v3/app/upgrades/v3"
```

Comment:
One of the things that will have to happen for the v4 upgrade is moving the go package to major version 4 in our go.mod and updating all the import references (`module/v3` becomes `module/v4`).

Reply:
But not in this PR right?

Reply:
You can open another PR after this if you want but it will be necessary for the v4 release.